code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -989,6 +989,17 @@
 	required_reagents = list(SODIUMCHLORIDE = 2, CLEANER = 2, OXYGEN = 1)
 	result_amount = 1
 
+//This one isn't even close the the real life reaction but will have to do to avoid conflicts with the above reactions.
+/datum/chemical_reaction/luminol
+	name = "Luminol"
+	id = LUMINOL
+	result = LUMINOL
+	required_reagents = list(CLEANER = 3, CARBON = 8)
+	result_amount = 10
```

Comment:
Your changelog doesn't match this, it still has the "realistic" results with byproduct water and hydrogen.

Reply:
OP updated.